src/main/java/uk/gov/hmcts/cmc/claimstore/config/DocAssemblyConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.cmc.claimstore.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
```

Comment:
we don't need this Configuration call, if it is just for retry, ConditionalOnProperty can be put on individual services ices as well.

Reply:
👍